07-advanced-git/README.md

```diff
@@ -1 +1,3 @@
-# WIP
+# Table of contents
+
+1. [`Git Log`](./07-01-git-log/README.md)
```

Comment:
```suggestion
1. [Git Log](./07-01-git-log/README.md)
```
![image](https://user-images.githubusercontent.com/34702921/87403263-38dbb580-c5ef-11ea-94a0-c0f5ad85e943.png)

Not sure if you want the root `README.md` to also follow the same format